ui/raidboss/popup-text.ts

```diff
@@ -921,7 +921,7 @@ export class PopupText {
 
     trigger.output = TriggerOutputProxy.makeOutput(
       trigger,
-      this.options.DisplayLanguage,
+      this.options.AlertsLanguage ?? this.options.DisplayLanguage,
```

Comment:
```suggestion
      this.displayLang,
```

I think this is meant to be the above, which includes even more fallback options.